src/development/LocalTest/Services/Storage/Implementation/DataRepository.cs

```diff
@@ -84,6 +84,12 @@ public Task<DataElement> Update(DataElement dataElement)
             string path = GetDataPath(dataElement.InstanceGuid, dataElement.Id);
             Directory.CreateDirectory(GetDataCollectionFolder());
             Directory.CreateDirectory(GetDataForInstanceFolder(dataElement.InstanceGuid));
+
+            using (StreamWriter sw = new StreamWriter(path, false))
+            {
+                sw.Write(dataElement.ToString());
+            }
+
             File.WriteAllText(path, dataElement.ToString());
```

Comment:
Wouldnt the file be written two times here?

Reply:
Yup!, deleting the line now ;)